lib/logstash/outputs/file.rb

```diff
@@ -42,6 +52,8 @@ class LogStash::Outputs::File < LogStash::Outputs::Base
   # into this file and inside the defined path.
   config :filename_failure, :validate => :string, :default => '_filepath_failures'
 
+  default :codec, "json_lines"
```

Comment:
I wonder why not just using json codec? is this option keeping backwards compatibility with previous behaviour? have you tested that?